lib/osf-components/addon/components/registries/schema-block-renderer/editable/radio/template.hbs

```diff
@@ -0,0 +1,19 @@
+<Registries::SchemaBlockRenderer::Editable::Base @helpText={{this.schemaBlock.helpText}}>
+    <FormControls
+        @changeset={{@changeset}}
+        @disabled={{@disabled}}
+        @shouldShowMessages={{@shouldShowMessages}}
+        ...attributes as |form|
+    >
+        <form.radio
+            @id={{@uniqueID}}
+            @valuePath={{@schemaBlock.registrationResponseKey}}
+            @options={{this.optionBlockValues}}
+            @onchange={{@onInput}}
+            as |option|
```

Comment:
Does `option` make sense here? Isn't this the whole control (which may have many options)?